src/Glpi/Form/Destination/CommonITILField/AssociatedItemsField.php

```diff
@@ -272,14 +273,22 @@ public function convertFieldConfig(FormMigration $migration, Form $form, array $
                         specific_associated_items: json_decode($rawData['associate_items'], true) ?? []
                     );
                 case 3: // PluginFormcreatorAbstractItilTarget::ASSOCIATE_RULE_ANSWER
+                    $mapped_item = $migration->getMappedItemTarget(
+                        'PluginFormcreatorQuestion',
+                        $rawData['associate_question']
+                    );
+
+                    if ($mapped_item === null) {
+                        $migration->result->addMessage(MessageType::Error, sprintf(
+                            "Question %d not found in a target form (%s)",
+                            $rawData['associate_question'],
+                            $form->getName()
+                        ));
+                    }
+
                     return new AssociatedItemsFieldConfig(
                         strategies: [AssociatedItemsFieldStrategy::SPECIFIC_ANSWERS],
-                        specific_question_ids: [
-                            $migration->getMappedItemTarget(
-                                'PluginFormcreatorQuestion',
-                                $rawData['associate_question']
-                            )['items_id']
-                        ],
+                        specific_question_ids: [$mapped_item['items_id'] ?? 0]
```

Comment:
Does this make sense ? Will this generate issues when the form will be used ?

IMHO, if a reference is broken, we should just not import this destination field config an output a generic error message like `sprintf(__('The "%s" destination field configuration of the form "%s" cannot be imported.'), $configurable_field->getLabel(), $form->getName())` should be logged. This can probably be done with a `try/catch` surrounding the `$configurable_field->convertFieldConfig()` call and an exception thrown when a required value is not found.